docs/configuration.rst

```diff
@@ -115,7 +115,7 @@ The final path of the file (including the repo name) should be ``<YOUR_ORG>/.git
 Mergeable will now use this file as the default when it cannot find one in a given
 repository or PR. It determines the file to use in the following order:
 
-1. A ``mergeable.yml`` inside the PR.
+1. A ``mergeable.yml`` inside the PR if the PR originates from the same repository. If it originates from a fork it is not loaded for security reasons.
```

Comment:
could you please add a link to the issue where the discussion is ? I think it would be nice if curious folks can look at the relevant discussion on `security reasons`